newspaper/parsers.py

```diff
@@ -211,28 +211,18 @@ def getText(cls, node):
 
     @classmethod
     def previousSiblings(cls, node):
```

Comment:
Confirmed the changes on this file are all proper via: http://lxml.de/api/lxml.etree._Element-class.html